client/my-sites/email/inbox/new-mailbox-upsell/index.tsx

```diff
@@ -43,9 +47,16 @@ const NewMailboxUpsell = ( { domains }: { domains: ResponseDomain[] } ) => {
 	return (
 		<div className="new-mailbox-upsell__container">
 			<div className="new-mailbox-upsell">
+				{ isFreeTrialNow && <Ribbon color="green">FREE</Ribbon> }
 				<div className="new-mailbox-upsell__messages">
 					<h2>{ translate( 'Need another mailbox?' ) }</h2>
-					<div>{ translate( 'Create a new one and activate it immediately.' ) }</div>
+					<div>
+						{ isFreeTrialNow
+							? translate(
+									'Create a new one free during your trial to experience multiple mailbox efficiency.'
```

Comment:
This is what we had in the asana card, but reading the sentence over again, it might sounds smoother with a `for` before `free`

>Create a new one `for` free during your trial to experience multiple mailbox efficiency.

Reply:
Done